lib/models/content_types.ts

```diff
@@ -13,6 +13,10 @@ export enum ContentType {
   Editor = 'SN|Editor',
   ActionsExtension = 'Extension',
   UserPrefs = 'SN|UserPreferences',
+  /**
+   * @deprecated Privileges are now inferred based on the user's active
+   * protections, like passcode or biometrics
+   */
   Privileges = 'SN|Privileges',
```

Comment:
Couldn't we just delete this content type?

Reply:
Probably should be deleted actually so that clients fail to compile rather than a warning. If they're using this they shouldn't be.

Reply:
I think so but I'm not completely sure. For now I'd like to keep it as documentation and make a decision later.

Reply:
GitHub is our source of documentation. I honestly don't think it's worth keeping around or postponing this.